uber/templates/registration/index.html

```diff
@@ -91,40 +127,42 @@
 </ol>
 
 <div class="row">
+    <div class="panel col-md-5">
+        <form role="form" method="get" action="index">
+            <input type="hidden" name="order" value="{{ order }}" />
+            <div class="input-group">
+                <input class="focus form-control" type="text" name="search_text" value="{{ search_text }}" />
+                <span class="input-group-btn">
+                    <button class="btn btn-default" type="submit">
+                        <span class="glyphicon glyphicon-search"></span>
+                    </button>
+                </span>
+            </div>
+        </form>
+    </div>
 
-<div class="panel col-md-5">
-<form role="form" method="get" action="index">
-<input type="hidden" name="order" value="{{ order }}" />
-<div class="input-group">
-<input class="focus form-control" type="text" name="search_text" value="{{ search_text }}" />
-<span class="input-group-btn"><button class="btn btn-default" type="submit">
-<span class="glyphicon glyphicon-search"></span>
-</button></span>
-</div>
-</form>
-</div>
```

Comment:
The rest of the changes to this file are just indentation fixes.  As you can see from the above, it's hard to see the structure of the DOM when nothing is indented.